vignettes/sjtbasic.Rmd

```diff
@@ -77,31 +77,16 @@ Most `sjt` function invisibly return a structure with following values:
 * the complete html-output `output.complete`
 * the html-table with inline-css for use with knitr `knitr`
 
-`page.style` and `page.content` are described below in the CSS section. If you want to include a HTML table in your knitr document, simply use an inline code chunk. Be sure to use `no.output=TRUE`, because else your output would also be opened in a viewer pane or webbrowser.
+`page.style` and `page.content` are described below in the CSS section. If you want to include a HTML table in your knitr document, simply use the corresponding function within a code chunk. Be sure to use `no.output=TRUE`, because else your output would also be opened in a viewer pane or webbrowser.
 
 `r rinline("sjt.frq(efc\\$e42dep, no.output=TRUE)\\$knitr")`
```

Comment:
Maybe the line around `rinline <- function(code){...` neither. I'll check that at home.